app/src/main/java/de/grobox/transportr/trips/search/DirectionsViewModel.kt

```diff
@@ -207,5 +209,6 @@ class DirectionsViewModel @Inject internal constructor(
             application.applicationContext, network.networkProvider,
             settingsManager, locationRepository, searchesRepository
         )
+        viaSupported = MutableLiveData(network.networkProvider.hasCapabilities(NetworkProvider.Capability.TRIPS_VIA))
```

Comment:
If this is static information known at the time of `init` and never changing, it wouldn't need to be a LiveData I guess, but its fine.

Reply:
You are right, it would not be necessary. But I guess it won't harm either and the advantage is to have only LiveData in the interface to the ViewModel.

Reply:
I don't see this as a big advantage, but sure it doesn't hurt.